library/src/main/java/com/novoda/downloadmanager/AndroidLocalFilesDirectory.java

```diff
@@ -0,0 +1,24 @@
+package com.novoda.downloadmanager;
+
+import android.content.Context;
+
+import java.util.Arrays;
+import java.util.List;
+
+class AndroidLocalFilesDirectory implements LocalFilesDirectory {
```

Comment:
This is just a simple wrapper around the `context`'s file operations, will be built on in subsequent PRs.